pypeit/traceslits.py

```diff
@@ -1158,7 +1169,7 @@ def save_master(self, root=None, gzip=True):
         ----------
         root : str (Optional)
           Path+root name for the output files
-        gzip : bool (optional)
+        gzip : bool (optional) JFH TURNED OFF GZIPPING
```

Comment:
Hmm..   These make for some ugly big files..

Reply:
Yeah, well none of our other masters gzipped. And the only reason why the files are ugly big is that you are writing out a bunch of junk that we don't need to the master file. I don't think the masters should contain the entire class. If you want to debug the class, just debug the class with a debugger.